sdk/sdk.go

```diff
@@ -31,7 +31,7 @@ type SDK struct {
 func New(m manager.Manager, c container.Container, db database.ServiceDB, instanceDB database.InstanceDB, execDB database.ExecutionDB) *SDK {
 	ps := pubsub.New(0)
 	return &SDK{
-		Service:   servicesdk.New(m, c, db, execDB),
+		Service:   servicesdk.New(m, c, db, instanceDB, execDB),
```

Comment:
Instance can know service but service cannot know about instance. This is following the data structure. Instance depends on service but service doesn't depends on instance so let's not create this dependency here. 

The api (that has access to service and instance) should do this check.

Reply:
done